### ISSUE TITLE
istio 1.2.2 ignore include/exclude port annotation in pod level

### ISSUE THREAD

**Issue by 1hanymhajna** [istio/istio#20307]
Hi, 
i'm using istio vesion 1.2.2, mtls feature is enabled,
As the documentation, i can exclude 
I tried to use this annotation but it it seems like istio ignore this annotation exclude specific inbound port from the mesh,
my global istio configuration is:
```
    includeInboundPorts: "*"
    excludeInboundPorts: ""
```

And want to exclude specific port in a specific pod ,
as the documentation I can use this annotation in deployment level:
```
 traffic.sidecar.istio.io/excludeInboundPorts: "80,8000"
```
**And this is the scenario that i tried:**
1- deploy a httpbin with this annotation
```
 traffic.sidecar.istio.io/excludeInboundPorts: "80,8000"
```
2- deploy sleep with this annotation:
traffic.sidecar.istio.io/includeOutbondPorts: "80,8000"

3- Send a request from sleep app to httpbin app

**expected behavior:**
To get an error cause sleep app pass the traffic via istio with mtls enable, so it should encrypt it, and httpbin app got the traffic on this port directly, to the httpbin container, so it should failed to decrypt the encrypted message it got

**Actual behavior:**
the request succeeded and ignoring the annotations

**Comment by howardjohn** [comment]
Please try on a newer version, this may have already been fixed: https://istio.io/news/support/announcing-1.2-eol/

**Comment by 1hanymhajna** [comment]
Thank you @howardjohn for your fast answer,
Can you please send the link to the relevant issue?

**Comment by howardjohn** [comment]
I am not sure which PR fixed it but I have verified its fixed at least in 1.6:

```
manifests/charts/istio-control/istio-discovery/files/injection-template.yaml
27-    - "-x"
28-    - "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/excludeOutboundIPRanges` .Values.global.proxy.excludeIPRanges }}"
29-    - "-b"
30-    - "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/includeInboundPorts` `*` }}"
31-    - "-d"
32:    - "15090,15021,{{ excludeInboundPort (annotation .ObjectMeta `status.sidecar.istio.io/port` .Values.global.proxy.statusPort) (annotation .ObjectMeta `traffic.sidecar.istio.io/excludeInboundPorts` .Values.global.proxy.excludeInboundPorts) }}"
33-    {{ if or (isset .ObjectMeta.Annotations `traffic.sidecar.istio.io/excludeOutboundPorts`) (ne (valueOrDefault .Values.global.proxy.excludeOutboundPorts "") "") -}}
34-    - "-o"
35-    - "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/excludeOutboundPorts` .Values.global.proxy.excludeOutboundPorts }}"
36-    {{ end -}}
37-    {{ if (isset .ObjectMeta.Annotations `traffic.sidecar.istio.io/kubevirtInterfaces`) -}}
```
